base/applications/taskmgr/lang/pt-PT.rc

```diff
@@ -0,0 +1,443 @@
+LANGUAGE LANG_PORTUGUESE, SUBLANG_PORTUGUESE
```

Comment:
SUBLANG_NEUTRAL

Reply:
com as alteraçoes que fiz gerou um arquivo .dccache devo fazer4 commit dele também?

Reply:
No. You must delete that file

Reply:
okay

Reply:
i add file to gitignore file for never commit this file

Reply:
Just out of curiosity, what software did created that `.dccache` file?

Reply:
vscode

Reply:
i don't know why i think is a temporary file

Reply:
https://stackoverflow.com/questions/66844803/how-do-i-prevent-the-dccache-file-from-being-created